scripts/kconfig/hardenconfig.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from kconfiglib import Symbol, standard_kconfig, standard_config_filename
+
+import csv
+import os
+
+
+def hardenconfig(kconf):
+    kconf.load_config()
+
+    hardened_kconf_filename = os.environ['ZEPHYR_BASE'] + os.sep +  \
+        "scripts" + os.sep + "kconfig" + os.sep + "hardened.csv"
+
+    options = compare_with_hardened_conf(kconf, hardened_kconf_filename, {})
+
+    display_results(options)
+
+
+class Option(object):
```

Comment:
Can do

```python
class Option:
```

Explicitly inheriting from `object` only makes a difference for Python 2.

There used to be a distinction between "old-style" and "new-style" classes in Python 2, where you needed to inherit from `object` to get a "new-style" class. Python 3 only has new-style classes though.